src/types/Options.ts

```diff
@@ -60,6 +60,24 @@ export type NamespaceResolver = (prefix: string) => string | null;
  * @public
  */
 export type Options = {
+	/**
+	 * WARNING: THIS OPTION ENABLES EXPERIMENTAL BEHAVIOR.
+	 *
+	 * What compiler backend to use. 
+	 *
+	 * Defaults to the stable 'expression' backend.
+	 *
+	 * FontoXPath has two backends: the older 'expression' backend and the
+	 * experimental 'js-codegen' backend. The js-codegen backend can be a lot
+	 * faster than the expression backend. If you want to try it out, use the
+	 * 'auto' option. This ensures that, if the js-codegen fails to compile a
+	 * given query, the expression backend is used instead. WARNING: this is
+	 * experimental.
+	 *
+	 * The 'js-codegen' option is used for TESTING ONLY. 
```

Comment:
Done, used @beta since @experimental was rejected by `npm run build`. 

> Warning: built/types/Options.d.ts:68:8 - (tsdoc-unsupported-tag) The TSDoc tag "@experimental" is not supported by this tool